.github/workflows/ci.yml

```
@@ -1,6 +1,6 @@
 # Used as inspiration: https://github.com/mvdan/github-actions-golang
 
-name: Cross-Platform
+name: Cross-Platform Tests
```

Idk I think we can shorten this frankly

```suggestion
name: Tests
```